fastlane/lib/fastlane/lane_manager.rb

```diff
@@ -173,15 +173,21 @@ def self.load_dot_env(env)
       return unless base_path
       require 'dotenv'
 
-      Actions.lane_context[Actions::SharedValues::ENVIRONMENT] = env if env
-
       # Making sure the default '.env' and '.env.default' get loaded
       env_file = File.join(base_path, '.env')
       env_default_file = File.join(base_path, '.env.default')
       Dotenv.load(env_file, env_default_file)
 
-      # Loads .env file for the environment passed in through options
-      if env
+      return unless envs
+
+      envs = envs.split(",") # multiple envs?
+
+      Actions.lane_context[Actions::SharedValues::ENVIRONMENT] = envs
+
+      # Loads .env file for the environment(s) passed in through options
+      envs = [envs] unless envs.kind_of? Array
+
+      envs.each do |env|
         env_file = File.join(base_path, ".env.#{env}")
```

Comment:
I'd break this out into a `load_env(env)` fn for readability, because right now this fn is getting pretty complex